frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -27,7 +31,7 @@ interface AggregationPickerProps {
   clauseIndex?: number;
   operators: Lib.AggregationOperator[];
   hasExpressionInput?: boolean;
-  onSelect: (operator: Lib.Aggregable) => void;
+  onSelect: (operator: Lib.Aggregable[]) => void;
```

Comment:
This is now an array, since the new shortcut can produce multiple aggregations at once.

Reply:
Reverting this because: https://metaboat.slack.com/archives/C0645JP1W81/p1717158820516559?thread_ts=1716988161.943959&cid=C0645JP1W81